lighthouse-core/scripts/compare-timings.js

```diff
@@ -19,26 +20,43 @@ const ROOT_OUTPUT_DIR = `${LH_ROOT}/timings-data`;
 const argv = yargs
   .help('help')
   .describe({
+    // common flags
     'name': 'Unique identifier, makes the folder for storing LHRs. Not a path',
+    'report-exclude': 'Regex of properties to exclude. Set to none to disable default',
     // --collect
     'collect': 'Saves LHRs to disk',
     'lh-flags': 'Lighthouse flags',
     'urls': 'Urls to run',
     'n': 'Number of times to run',
     // --summarize
     'summarize': 'Prints statistics report',
-    'measure-filter': 'Regex filter of measures to report. Optional',
+    'measure-filter': 'Regex of measures to include. Optional',
     'output': 'table, json',
+    // --compare
+    'compare': 'Compare two sets of LHRs',
+    'delta-sort': 'Property to sort by its delta',
```

Comment:
wdyt about `sort-property`?

Reply:
b/c that implies you provide the entire property name, and do you wanna google "unicode delta" to do that? :P

maybe `property-delta-sort`? `delta-property-sort`? ... how does yargs do shorthand flags?

Reply:
you can `.alias` it to a single letter, so I tend to prefer verbose full names. if we print the unicode delta in our default `mean Δ` then folks can copy/paste :)

`delta-property-sort` sgtm 👍